platform/ui/src/components/StudyListPagination/StudyListPagination.js

```diff
@@ -2,27 +2,28 @@ import React, { useState } from 'react';
 import PropTypes from 'prop-types';
 import { Button, ButtonGroup, Typography, Select } from '../';
 import { useTranslation } from 'react-i18next';
+import ConfigPoint from 'config-point';
 
-const StudyListPagination = ({
+export function StudyListPagination({
```

Comment:
Shows a simple config-point extension for this item.  That allows customizing the pagination items.